pkgs/servers/mail/listmonk/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, buildGoModule, fetchFromGitHub, callPackage, stuffbin }:
+let
+  frontend = callPackage ./frontend.nix { };
+  vfsMappings = [
+    "config.toml.sample"
+    "schema.sql"
+    "queries.sql"
+    "static/public:/public"
+    "static/email-templates"
+    "${frontend}:/admin"
+    "i18n:/i18n"
+  ];
+  putIntoBinary = mapping: ''
```

Comment:
We should move this into a let in directly where it is used.

Reply:
Done.